.github/workflows/cloud-deploy-reusable-app-service-deploy.yml

```diff
@@ -0,0 +1,217 @@
+    if: inputs.service_name != 'worker'
+    needs:
+      - infrastructure_data
+    runs-on: ubuntu-latest
+    timeout-minutes: 80
+    environment: ${{ inputs.environment }}
+    permissions:
+      contents: read
+      deployments: write
+    steps:
+      - run: echo "Deploying ${{ inputs.service_name }} to ${{ vars.TF_WORKSPACE }} And Docker Tag ${{ inputs.docker_image }}"
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ needs.infrastructure_data.outputs.aws_region }}
+
+      - name: Download task definition
+        run: |
+          aws ecs describe-task-definition --task-definition ${{ needs.infrastructure_data.outputs.task_name }} \
+          --query taskDefinition > task-definition.json
+
+      - name: Render Amazon ECS task definition
+        id: render-web-container
+        uses: aws-actions/amazon-ecs-render-task-definition@39c13cf530718ffeb524ec8ee0c15882bcb13842
+        with:
+          task-definition: task-definition.json
+          container-name: ${{ needs.infrastructure_data.outputs.ecs_container_name }}
+          image: ${{ inputs.docker_image }}
+
+      - name: Deploy to Amazon ECS service
+        uses: aws-actions/amazon-ecs-deploy-task-definition@3e7310352de91b71a906e60c22af629577546002
+        with:
+          task-definition: ${{ steps.render-web-container.outputs.task-definition }}
+          service: ${{ needs.infrastructure_data.outputs.ecs_service }}
+          cluster: ${{ needs.infrastructure_data.outputs.ecs_cluster }}
+          wait-for-service-stability: true
+
+  post-deploy:
+    needs:
+      - infrastructure_data
+      - deploy_service
+      - deploy_service_worker
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      deployments: read
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: get-npm-version
+        if: ${{ vars.DEPLOY_SENTRY_RELEASE }}
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@main
+        with:
+          path: apps/${{ inputs.sentry_project }}
+
+      - name: Create Sentry release
+        if: ${{ vars.DEPLOY_SENTRY_RELEASE }}
+        uses: getsentry/action-release@v1
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_ORG: novu-r9
+          SENTRY_PROJECT: ${{ inputs.sentry_project }}
+        with:
+          version: ${{ steps.package-version.outputs.current-version}}
+          version_prefix: v
+          environment: prod
+          ignore_empty: true
+          ignore_missing: true
+
+      - name: Set New Relic Region
+        if: ${{ inputs.New_Relic_GUID }}
+        id: set_region
+        run: |
+          if [[ "${{ env.aws_region }}" == *"us"* ]]; then
+            echo "New_Relic_Region=US" >> ${GITHUB_ENV}
```

Comment:
Please convert this to a NEW_RELIC_REGION env variable to remove this conditional based on region